edit-hopper.php

```diff
@@ -35,9 +35,19 @@ public function __construct() {
 		}
 
 		function init($screen) {
+			$box_title = "Edit something";
+                        switch ($screen) {
+                            case "post":
+                                $box_title = "Edit Posts";
+                                break;
+                            case "page":
+                                $box_title = "Edit Pages";
+                            default:
+                                break;
+                        }
 			add_meta_box(
 		        'edit_hopper_box',
-		        __( 'Edit Pages', 'edit_hopper_textdomain' ),
+		        __( $box_title, 'edit_hopper_textdomain' ),
```

Comment:
I'm pretty sure we can just use the variable **$screen** right here and then skip the entire case switch.